Dockerfile.e2e

```diff
@@ -1,6 +1,9 @@
-FROM openshift/origin-release:golang-1.10 as build
+FROM openshift/origin-release:golang-1.10
 
-RUN INSTALL_PKGS="curl python PyYAML" && \
+COPY hack/faq.repo /etc/yum.repos.d/ecnahc515-faq-epel-7.repo
```

Comment:
consistency with other places this is mentioned -- add `# our copy of faq and jq`?

Reply:
Fixed